### PR TITLE
URS-963: Move CI to GitHub Actions

### PR DESCRIPTION
**Pull request by andrewbenedictwallace**
- Creates Github action to run CI suite
- Removes all CI from .travis.yml - leaving only code to trigger jenkins builds

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@b818bea8cf015031920150a891dbdd4105cc7e47
+      with:
+        ruby-version: '2.5'
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+    - name: erblint
+      run: bundle exec erblint --lint-all
```
Comment (pghorpade) [inline .github/workflows/ci.yml:24]: should this be bundle exec rubocop

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+  rspec:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - name: Start docker services
```
Comment (pghorpade) [inline .github/workflows/ci.yml:46]: Does this need docker login too?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (andrewbenedictwallace) [reply .github/workflows/ci.yml]: it should!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (andrewbenedictwallace) [reply .github/workflows/ci.yml:44]: it does!

**Review by pghorpade — APPROVED** [review]
💯

**Pull request merged**